amazonia/classes/asg.py

```diff
@@ -62,7 +65,7 @@ def __init__(self, title, vpc, template, minsize, maxsize, subnets, load_balance
 
     def create_asg(self, title, minsize, maxsize, subnets, load_balancer, keypair, image_id, instance_type,
                    health_check_grace_period, health_check_type, iam_instance_profile_arn, userdata, sns_topic_arn,
-                   sns_notification_types):
+                   sns_notification_types, hdd_size):
```

Comment:
Sorry I know its not your code, but could you update the link in the docstring from http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-ec2-security-group.html  to http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-as-group.html